tabs/osd.js

```diff
@@ -463,6 +473,11 @@ OSD.chooseFields = function () {
           ]);
         }
       }
+      if (semver.gte(CONFIG.flightControllerVersion, "3.2.0")) {
+        OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
```

Comment:
A lot of features and changes get shared between Betaflight and cleanflight these days, and the goal is to be able to do so by cherry-picking commits without having to manually add changes (which will generate more work down the track because they often lead to merge conflicts). Since `flightControllerVersion` (i.e. the release number of the firmware) are not in synch between Betaflight and cleanflight, using it to check for the existence of a feature will require a manual edit when pulling a change across, so it should be avoided.

Thanks for changing.